### DISCUSSION TITLE
Parental option

### DISCUSSION THREAD

**Discussion by MKBontwikkeling** [SkyTubeTeam/SkyTube#1198] [Q&A]
Hi, I love the implementation of white and blacklisting etc. As a parent I would like to use it for my kids, but without password protection of the settings it is kind of useless in that way. Would it be possible to password protect the settings menu?

Thanks in advance!

**Comment by DahilerMehvOlur** [comment]
Oh man you read my mind. Yeah I also want to add password for to enter blacklist. My child knows that I am blocking some channels. 😃
Bu I think we need open this request in "feature request" section.

**Reply by gzsombor** [reply]
Yes, that would be a great feature :)
